server/api.js

```diff
@@ -30,10 +31,10 @@ router.get("/laptop_request", async (req, res) => {
 
 		const laptopRequests = result.rows.map((row) => {
 			return {
-				firstName: row.firstname,
-				lastName: row.lastname,
+				firstName: row.firstName,
```

Comment:
Changing capitalisation here shouldn't be necessary - fields from a PostgreSQL database will have lowercase names in almost all cases.

Reply:
ok i will do